package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "integration-ts-example",
+    "type": "module",
+    "version": "1.0.0",
+    "description": "This example demonstrates how to use the Integration Node.js library.",
+    "main": "light.js",
+    "dependencies": {
+        "integration-api": "ZsakaiBalint/integration-node-library#iss-27"
```

Comment:
This branch has the `./dist` folder commited.
There should be a way to avoid this.